client/dashboard/plugins/manage/index.tsx

```diff
@@ -49,15 +50,14 @@ export default function PluginsList() {
 			enableGlobalSearch: true,
 		} ) );
 	}, [] );
-	const { data: plugins } = useMemo(
-		() =>
-			filterSortAndPaginate(
-				mapApiPluginsToDataViewPlugins( sitesById, sitesPlugins ),
-				view,
-				fields
-			),
-		[ sitesById, sitesPlugins, fields ]
-	);
+	const { data: plugins, paginationInfo } = useMemo( () => {
+		const result = filterSortAndPaginate(
```

Comment:
Nit: I think we can just return immediately as before, instead of introducing an intermediate `result` variable which does nothing here. (Basically we can revert this particular change)